pages/company.tsx

```diff
@@ -31,82 +44,123 @@ export default function Company() {
-                          <dd className="mt-1 text-sm text-gray-900">{company?.primary_email}</dd>
-                        </div>
-                        <div className="sm:col-span-1">
-                          <dt className="text-sm font-medium text-gray-500">Primary Phone Number</dt>
-                          <dd className="mt-1 text-sm text-gray-900">{company?.primary_phone_number}</dd>
-                        </div>
-                        <div className="sm:col-span-1">
-                          <dt className="text-sm font-medium text-gray-500">Departments</dt>
-                          <>
-                            {company.departments.map(dept => (
-                              <dd className="mt-1 text-sm text-gray-900">{dept.name}</dd>
-                            ))}
-                          </>
-                        </div>
-                        <div className="sm:col-span-1">
-                          <dt className="text-sm font-medium text-gray-500">Locations</dt>
-                          <>
-                            {company.locations.map(location => (
-                              <>
-                                <dd className="mt-1 text-sm text-gray-900">{location.line1}</dd>
-                                <dd className="mt-1 text-sm text-gray-900">{location.line2}</dd>
-                                <dd className="mt-1 text-sm text-gray-900">{location.city}, {location.state} {location.postal_code}</dd>
-                                <dd className="mt-1 text-sm text-gray-900 mb-6">{location.country}</dd>
-                              </>
-                            ))}
-                          </>
-                        </div>
-                        <div className="sm:col-span-1">
-                          <dt className="text-sm font-medium text-gray-500">Accounts</dt>
-                          <>
-                            {company.accounts.map(account => (
-                              <>
-                                <dd className="mt-1 text-sm text-gray-900">Institution Name: {account?.institution_name ?? 'null'}</dd>
-                                <dd className="mt-1 text-sm text-gray-900">Account Name: {account?.account_name ?? 'null'}</dd>
-                                <dd className="mt-1 text-sm text-gray-900">Account Type: {account?.account_type ?? 'null'}</dd>
-                                <dd className="mt-1 text-sm text-gray-900">Routing Number: {account?.routing_number ?? 'null'}</dd>
-                                <dd className="mt-1 text-sm text-gray-900 mb-6">Account Number: {account?.account_number ?? 'null'}</dd>
-                              </>
-                            ))}
-                          </>
-                        </div>
+        <div style={{
+            display: 'flex',
+            alignItems: 'right',
+            justifyContent: 'right',
+        }}>
+          <ToggleButtonGroup
```

Comment:
The Toggle button font is not the same as the rest of the page text. I believe the toggle button component uses Roboto, and the rest of the page uses Segoe UI. Can the toggle component inherit the standard font?